src/maintainer/knowledge_base.rst

```diff
@@ -118,28 +118,56 @@ use a general web search to explore — while these topics are beyond the
 scope of this documentation, there are ample discussions on them on the broader
 Internet.
 
+To bootstrap a conda environment and install ``conda-build``, consider
+`miniforge <https://github.com/conda-forge/miniforge>`__.
+
+
+Executing a build
+^^^^^^^^^^^^^^^^^
+
+The ``build-locally.py`` script does not support Windows (yet, PRs welcome!).
+You can use ``conda build recipe/ -m .ci_support/choose_your_config.yaml`` as
+a workaround for now.
+
+Using a local channel looks a bit different in Windows, too (note the
+triple-forward-slash in the channel name):
+
+.. code-block::
+
+    conda create -n my-new-env -c file:///C:\Users\User\miniforge3\conda-build my-package
```

Comment:
I've had no problems using unmodified local paths as channels if they're escaped by quotes, i.e.
```
conda create -n my-new-env -c "C:\Users\me\path\to\wherever\I\unpacked\an\artefact"
```
I think this is easier than triple slashes resp. mixing between forward/backward slashes?

Reply:
Thanks, that is much easier :P I don't know Windows well so I struggled a bit with this when testing on a VM. Will update shortly...

Reply:
60c06a8f